corehq/apps/users/models.py

```diff
@@ -78,11 +81,34 @@
 from six.moves import range
 from six.moves import map
 
+debug_save_logger = logging.getLogger('debug_user_save')
+
 COUCH_USER_AUTOCREATED_STATUS = 'autocreated'
 
 MAX_LOGIN_ATTEMPTS = 5
 
 
+def log_user_save(type_, doc):
```

Comment:
Is there a reason to include this in master? Seems like it'll only affect local testing for a short while (until this is resolved) so it could be in a branch

Reply:
Actually I was wanted this deployed to ICDS. I've tried to repro the issue locally and on other envs and haven't been able to.

Reply:
so you'll change https://github.com/dimagi/commcare-hq/pull/18828/files#diff-bdf3ecebd8379ca98cc89e545fc90899R1289 locally on a webworker on ICDS to actually get it to log? makes sense if that's the way forward